starter-code/server.js

```diff
@@ -1,18 +1,28 @@
 'use strict';
 
-// TODO: Initialize your project using NPM to create and populate a package.json file
+//DONE: Initialize your project using NPM to create and populate a package.json file
 
-// TODO: Require the Express package that you installed via NPM, and instantiate the app
+//DONE: Require the Express package that you installed via NPM, and instantiate the app
 // Remember to install express, and be sure that it's been added to your package.json as a dependency
 // There is also a package here called body-parser, which is required in for use in a new route.
 // Be sure to install that and save it as a dependency after you create your package.json.
 const bodyParser = require('body-parser').urlencoded({extended: true});
 const PORT = process.env.PORT || 3000;
+const express = require('express');
+const app = express();
 
-// TODO: Include all of the static resources as an argument to app.use()
+// DONE: Include all of the static resources as an argument to app.use()
+app.use(express.static('./public'));
+// DONE: (STRETCH) Write a new route that will handle a request and send the new.html file back to the user
 
-// TODO: (STRETCH) Write a new route that will handle a request and send the new.html file back to the user
+app.get('/', function(request, response) {
```

Comment:
Need to make this to a separate url route so that the index .html doesnt get fired by the base path. /new would be appropriate